tests/testPsfDetermination.py

```diff
@@ -219,7 +219,7 @@ def tearDown(self):
         del self.catalog
 
     @staticmethod
-    def setupDeterminer(exposure, nEigenComponents=3, starSelectorAlg="secondMoment"):
+    def setupDeterminer(exposure, schema, nEigenComponents=3, starSelectorAlg="secondMoment"):
```

Comment:
Done.  I just had to do something very similar to the tests in meas_extensions_psfex, so this was pretty straightforward.